README-develop.md

```diff
@@ -20,6 +15,15 @@ open http://$(docker-compose port nginx 80)
 
 ## Setup for development
 
+Add core-bundle to the project:
```

Comment:
Please add comment about not committing composer.json after this change has been made.